cloudbuild.yaml

```diff
@@ -49,13 +49,9 @@ steps:
 - name: 'gcr.io/$PROJECT_ID/e2e-go-mod-ginkgo:0.4.28'
   entrypoint: 'bash'
   env:
-  - 'ACK_GINKGO_RC=true'
-  - 'ACK_GINKGO_DEPRECATIONS=1.16.5'
-  - 'GOLANG_PROTOBUF_REGISTRATION_CONFLICT=warn'
   - 'RUN_KUBE_TESTS=1'
   - 'RUN_CONSUL_TESTS=1'
   - 'RUN_VAULT_TESTS=1'
-  - 'TAGGED_VERSION=$TAG_NAME'
```

Comment:
I just moved the TAGGED_VERSION env var, from being defined on many steps, to being defined at the job level